### DISCUSSION TITLE
Styling Handbook link in react.fluentui.dev site broken

### DISCUSSION THREAD

**Discussion by MChez** [microsoft/fluentui#31322] [General]
Hi,

Just a quick note to let you know that the 'Styling handbook' link at the top of the Developer > [Styling Components page](https://react.fluentui.dev/?path=/docs/concepts-developer-styling-components--page) in the https://react.fluentui.dev site is broken.

I think the new URL is https://github.com/microsoft/fluentui/blob/master/docs/react-v9/contributing/rfcs/react-components/styles-handbook.md

**Comment by khmakoto** [comment]
I think this has been fixed now!
